trinity/algorithm/policy_loss_fn/ppo_policy_loss.py

```diff
@@ -50,6 +52,11 @@ def __call__(  # type: ignore
         **kwargs,
     ) -> Tuple[torch.Tensor, Dict]:
         negative_approx_kl = logprob - old_logprob
+        if self.fallback_to_policy_gradient:
+            print('------')
```

Comment:
Use `trinity.utils.log.get_logger` instead of `print`, or just remove these lines